sample-skill/lambda/lambda_function.py

```diff
@@ -82,19 +84,19 @@ def get_course_progress(username: str, course_id: str, token: str) -> float | No
         token (str): The Bearer token used to consume the API.
 
     Returns:
-        float | None: The progress of the student in the course as a percentage
-        (0.00 - 100.00), or None if the progress can't be retrieved.
+        float: The progress of the student in the course as a percentage
+        (0.00 - 100.00), or 0.0 if the progress can't be retrieved.
     """
     endpoint_url = f"{API_DOMAIN}/eox-core/api/v1/grade/"
     payload = {"username": username, "course_id": course_id}
     headers = {"Authorization": f"Bearer {token}"}
 
     response = make_request(endpoint_url, data=payload, headers=headers)
 
-    return round(response["earned_grade"]*100, 2) if response else None
+    return round(response.get("earned_grade", 0) * 100, 2)
```

Comment:
Why *100?

Reply:
Because the endpoint response is a float of type `0.xx`